torch/_subclasses/fake_tensor.py

```diff
@@ -332,8 +332,13 @@ def from_real_tensor(
         dynamic_dims: "Optional[DimList[DimDynamic]]" = None,
         constraint_dims: "Optional[DimList[DimConstraint]]" = None,
         memoized_only=False,
+        force_fresh=False,
     ):
-        maybe_memo = self._get_memo(t)
+        if force_fresh:
```

Comment:
I'm a bit worried about this setting... is there a way to get the memoized, but with a modified shape/stride meta and increased _version?

Reply:
Does modifying run into some safety checks? Maybe we ought to be able to disable the safety checks as needed.

Reply:
See my comment below.

Reply:
> I'm a bit worried about this setting... (for object identity ressons) is there a way to get the memoized, but with a modified shape/stride meta and increased _version?

Well, the problem here is that you need to access its original value - while we do want memo to give us the same fake tensor, which should be kept up to date as we trace/prop. Bypassing memo is a hack, I'm going to jam on this w/ @ezyang today.

> Does modifying run into some safety checks? Maybe we ought to be able to disable the safety checks as needed. (if we know we are passing a symnode)

What do you mean?